src/ui/public/notify/banners/banners.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Banners represents a prioritized list of displayed components.
+ */
+export class Banners {
```

Comment:
Nitpick, but how about we name this `TopBanners` and rename the file `top_banners.js`? This will make it a bit easier to find this file when people are trying to figure out where `topBanners` is coming from. If we decide to create bottom banners at some point then I think we can rename & abstract away a base `Banners` class, but I'd prefer to do that when the time comes.

Reply:
Changing `topBanners` to `banners` and thus keeping this.